<?xml version="1.0" ?>

<launch>

  <arg name="map_file" default="$(find tb_waiter)/maps/lab2.yaml"/>
  <arg name="rviz_robot" default="robot1"/>
  
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" > 
  </node>
  
  <!-- move_base global
  <include file="$(find tb_waiter)/launch/simulation/stage/include/move_base_stage.launch.xml" />  -->
  
  <!-- BEGIN ROBOT 0-->
  <group ns="robot_0">
    <param name="tf_prefix" value="robot_0" />
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" /> 
    <node pkg="tb_waiter" name="fake_localization" type="fake_amcl" output="screen"/>
    <include file="$(find tb_waiter)/launch/simulation/stage/include/move_base_stage.launch.xml" />
  </group>
 
  <!-- BEGIN ROBOT 1-->
  <group ns="robot_1">
    <param name="tf_prefix" value="robot_1" />
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" /> 
    <node pkg="tb_waiter" name="fake_localization" type="fake_amcl" output="screen"/>
    <include file="$(find tb_waiter)/launch/simulation/stage/include/move_base_stage.launch.xml" />
  </group>
    
</launch>

      
